src/components/Quizzes.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import {
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    CardMedia,
+    Typography
+} from '@mui/material';
+import RadioButtonsGroup from './RadioButtonsGroup';
+import BaseModal from './BaseModal';
+import DescriptionAlerts from './DescriptionAlert';
+
+const CardStyle = styled(Card)(() => ({
+    position: 'relative',
+    maxWidth: '350px',
+    height: '100%',
+    padding: '10px',
+    fontWeight: 'bold',
+}));
+
+function Quizzes({ card, id }) {
+    const typeQuiz = {
```

Comment:
If you use it not in one place
Add constant folder and c eats file here either type and make export in place where you will need